spark3-extensions/src/main/scala/org/apache/spark/sql/catalyst/optimizer/RewriteMergeInto.scala

```diff
@@ -78,8 +78,8 @@ case class RewriteMergeInto(spark: SparkSession) extends Rule[LogicalPlan] with
         val joinPlan = Join(source, targetTableScan, LeftAnti, Some(cond), JoinHint.NONE)
 
         val mergeParams = MergeIntoParams(
-          isSourceRowNotPresent = FALSE_LITERAL,
```

Comment:
Will it be easier to reason about this logic in terms of `noMatchInSourceRelation`/`noMatchInTargetRelation`?

Reply:
Negating the flag reads better to me, personally. `rowPresentInSouce`  or `isRowPresentInSource` ?

Reply:
I agree with using positive flags instead of negative ones. Negatives are harder to reason about when using them because you may need to negate again: `!noMatchInSource` is harder than `isPresentInSource`